relay-dynamic-config/src/metrics.rs

```diff
@@ -114,7 +114,8 @@ pub struct CustomMeasurementConfig {
 ///      - Emit a `usage` metric and use it for rate limiting.
 ///      - Delay metrics extraction for indexed transactions.
 ///  - 4: Adds support for `RuleConfigs` with string comparisons.
-const TRANSACTION_EXTRACT_VERSION: u16 = 4;
+///  - 5: No change, bumped together with [`MetricExtractionConfig::MAX_SUPPORTED_VERSION`].
+const TRANSACTION_EXTRACT_VERSION: u16 = 5;
```

Comment:
Should we also rename this to include `max supported` somewhere?

```suggestion
const TRANSACTION_EXTRACTION_MAX_SUPPORTED_VERSION: u16 = 5;
```